src/Exceptions/InvalidValueException.php

```diff
@@ -1,19 +0,0 @@
-<?php
-
-namespace Spatie\Enum\Exceptions;
-
-use InvalidArgumentException;
-
-class InvalidValueException extends InvalidArgumentException
```

Comment:
I would prefer this explicit exception over a type error if a non string is written into `EnumDefinition::$label`

Reply:
My preference is actually to use the type system to its best extent where possible.